### DISCUSSION TITLE
BRouter: via 6 position not mapped ...

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#32] [Q&A] [answered]
Trying waypoints 18 and 19 to connect by beeline I get this error message:
![IMG-20230209-WA0000](https://user-images.githubusercontent.com/121806238/217750716-d2e6c501-06a1-4d91-8c95-f8a94474e4e4.jpg)
I've installed south of Portugal in BRouter - offline planning works.
The route is part of the TET PT - unfortunately it was raining yesterday - so we couldn't test the beeline either.

**Comment by devemux86** [comment, marked as answer]
Thanks for the report!

This message comes from the BRouter app.
It means that the indicated via / shaping point 6 cannot be mapped / snapped onto the nearby road network.

Have you tried to zoom in and move the via / shaping 6 closer to the road network?

Is the route available to test it? What routing options?
Does it work on [BRouter website](https://brouter.de/brouter-web)?

**Comment by WolfgangZ42** [comment]
Thanks for the hint!
It seems that BRouter doesn't have the small pathes from the TET in its datafile. I'd to delete quite a lot of shaping points to finaly establish the
beeline:
![IMG-20230209-WA0001](https://user-images.githubusercontent.com/121806238/217773966-048c9612-6be0-422b-8b11-cc629b33252e.jpg)
So I set the beeline in kurviger.de and exported the file ones more. 
![IMG-20230209-WA0002](https://user-images.githubusercontent.com/121806238/217774965-010f7093-19b9-42d7-ab29-e37c445b596a.jpg)

**Comment by devemux86** [comment]
Or the selected BRouter profile cannot map the waypoint.
Each BRouter profile behaves differently with road paths.

What BRouter profile or Cruiser routing settings do you use?

**Reply by WolfgangZ42** [reply]
![Screenshot_20230209-095740](https://user-images.githubusercontent.com/121806238/217780931-146dbf73-35e6-4352-b7c1-5b3c9b30f18e.png)

I can't test much - have only my smartphone with me. The route: https://kurv.gr/hRBR9
